app/models/ability.rb

```diff
@@ -29,5 +29,15 @@ def initialize(user)
     else
       can :view, :all
     end
+
+    #check if user's chapter is active
+    #if not, only let them see Dashboard with information about deactivation
+    @user_chapter = Chapter.where(id: user.chapter_id)
+    @user_chapter.each do |c|
+      if c.is_active != 1
```

Comment:
GAH! THANK YOU! I didn't understand why that wasn't working for me? I only know to do the ![variable] approach, but is it because it's not a boolean? Ruby not a fan? I get so tangled in language syntax sometimes...